modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -801,26 +801,25 @@ Available resources vary between clusters. The number of possible clusters withi
 [discrete]
 [id="installation-vsphere-installer-infra-requirements-networking_{context}"]
 == Networking requirements
-Use Dynamic Host Configuration Protocol (DHCP) for the network and ensure that the DHCP server is configured to provide persistent IP addresses to the cluster machines.
+
+You can use Dynamic Host Configuration Protocol (DHCP) for the network and configure the DHCP server to set persistent IP addresses to machines in your cluster. In the DHCP lease, you must configure the DHCP to use the default gateway.
 
 [NOTE]
 ====
 You do not need to use the DHCP for the network if you want to provision nodes with static IP addresses.
 ====
 
-Configure the default gateway to use the DHCP server. All nodes must be in the same VLAN. You cannot scale the cluster using a second VLAN as a Day 2 operation.
-
-You must use the Dynamic Host Configuration Protocol (DHCP) for the network and ensure that the DHCP server is configured to provide persistent IP addresses to the cluster machines. In the DHCP lease, you must configure the DHCP to use the default gateway. All nodes must be in the same VLAN. You cannot scale the cluster using a second VLAN as a Day 2 operation.
+Provided you properly set firewall rules for your network, you can specify nodes on different VLANs.
```

Comment:
Soft suggestion for your consideration to replace "provided" with something like "if" or "assuming". There's not really an issue with it as-is. However, the "Be clear" goal in [this page](https://www.ibm.com/docs/en/ibm-style?topic=medium-conversational-style) of the ISG mentions to "use simple language" and in my (not strongly held) opinion, I feel like these alternatives would be somewhat simpler.
```suggestion
If you properly set firewall rules for your network, you can specify nodes on different VLANs.
```

Or even simpler might be to start with the optional thing and then mention the condition:
```suggestion
You can specify nodes on different VLANs, if you properly set firewall rules for your network.
```

Not sure if these are valuable suggestions, but just something to consider.